README.md

```diff
@@ -1,8 +1,43 @@
-# Northcoders News API
+# News API
 
-In order for this program to run, should you fork it, two .env files should be created locally - .env.development and .env.test
+---
 
+**Info**
 
+A REST API with the purpose of accessing application data.
+The intention here is to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture. 
```

Comment:
What kind of data are we expecting to be given here? I would recommend describing the kind of data the data base holds as well